src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { Form, Field } from "react-final-form";
+import { useNavigate, Link } from "react-router-dom";
+import "./LoginForn.scss";
+
+const LoginForm = ({ errorMessage, setErrorMessage }) => {
+  const navigate = useNavigate();
+  const validate = (values) => {
+    const errors = {};
+    if (!values.userName) {
+      errors.userName = "User Name Required";
+    }
+    if (!values.password) {
+      errors.password = "Password Required";
+    }
+    return errors;
+  };
+
+  const handleLoginSubmit = (values) => {
+    const registeredUsers = JSON.parse(localStorage.getItem("users"));
+    const user = registeredUsers?.find(
+      (user) => user?.userName === values?.userName
```

Comment:
If a user is available, his userName is also available right?
```suggestion
      (user) => user.userName === event.userName
```